src/test/java/com/google/api/codegen/testdata/java_test_library.baseline

```diff
@@ -109,19 +129,42 @@ public class LibraryServiceTest {
     Assert.assertEquals(1, actualRequests.size());
     CreateShelfRequest actualRequest = (CreateShelfRequest)actualRequests.get(0);
 
-    Assert.assertEquals(actualRequest.getShelf(), shelf);
+    Assert.assertEquals(shelf, actualRequest.getShelf());
+
+    Assert.assertEquals(expectedResponse.getName(), actualResponse.getName());
+    Assert.assertEquals(expectedResponse.getTheme(), actualResponse.getTheme());
+    Assert.assertEquals(expectedResponse.getInternalTheme(), actualResponse.getInternalTheme());
```

Comment:
The response should have its own equals() method, so I think you should be able to just compare expectedResponse & actualResponse.